Fix istio control and data planes naming in documentation

toffentoffen opened the pull request:
<!--
    ⚠️ Please make sure to read this template first, pull requests that don't accord with this template
    maybe closed without notice.
    Texts surrounded by `<` and `>` are meant to be replaced by you, e.g. <framework name>, <issue number>.
    Put an `x` in the `[ ]` to mark the item as CHECKED. `[x]`
-->

<!-- ==== 🐛 Remove this line WHEN AND ONLY WHEN you're fixing a bug, follow the checklist 👇 ====
### Fix <bug description or the bug issue number or bug issue link>
- [ ] Add a unit test to verify that the fix works.
- [ ] Explain briefly why the bug exists and how to fix it.
     ==== 🐛 Remove this line WHEN AND ONLY WHEN you're fixing a bug, follow the checklist 👆 ==== -->

<!-- ==== 📈 Remove this line WHEN AND ONLY WHEN you're improving the performance, follow the checklist 👇 ====
### Improve the performance of <class or module or ...>
- [ ] Add a benchmark for the improvement, refer to [the existing ones](https://github.com/apache/skywalking/blob/master/apm-commons/apm-datacarrier/src/test/java/org/apache/skywalking/apm/commons/datacarrier/LinkedArrayBenchmark.java)
- [ ] The benchmark result.
```text
<Paste the benchmark results here>
```
- [ ] Links/URLs to the theory proof or discussion articles/blogs. <links/URLs here>
     ==== 📈 Remove this line WHEN AND ONLY WHEN you're improving the performance, follow the checklist 👆 ==== -->

<!-- ==== 🆕 Remove this line WHEN AND ONLY WHEN you're adding a new feature, follow the checklist 👇 ====
### <Feature description>
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [ ] Update the documentation to include this new feature.
- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.
     ==== 🆕 Remove this line WHEN AND ONLY WHEN you're adding a new feature, follow the checklist 👆 ==== -->

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

wu-sheng reviewed (changes requested):
Hi, Marc, I think you mis-change the submodules accidentally. Please make sure they are unchanged.

![image](https://user-images.githubusercontent.com/5441976/150883476-374d9504-b1e4-4c04-9eec-6261711261ae.png)

toffentoffen commented:
> Hi, Marc, I think you mis-change the submodules accidentally. Please make sure they are unchanged.
> 
> ![image](https://user-images.githubusercontent.com/5441976/150883476-374d9504-b1e4-4c04-9eec-6261711261ae.png)

yes sorry, you are totally right. My bad. Will fix it right away.

I don't know how I got those changes, I haven't worked with submodules allot.

```
$ git fetch upstream master
remote: Enumerating objects: 1048, done.
remote: Counting objects: 100% (1048/1048), done.
remote: Compressing objects: 100% (497/497), done.
remote: Total 1048 (delta 322), reused 942 (delta 269), pack-reused 0
Receiving objects: 100% (1048/1048), 306.86 KiB | 1.31 MiB/s, done.
Resolving deltas: 100% (322/322), completed with 32 local objects.
From github.com:apache/skywalking
 * branch                master     -> FETCH_HEAD
   62ef59466..2d13c8afd  master     -> upstream/master
Fetching submodule apm-protocol/apm-network/src/main/proto
git reFrom github.com:apache/skywalking-data-collect-protocol
   fbbe955..667134d  master     -> origin/master
 * [new tag]         v8.9.0     -> v8.9.0
 * [new tag]         v8.9.1     -> v8.9.1
Fetching submodule oap-server/server-query-plugin/query-graphql-plugin/src/main/resources/query-protocol
From github.com:apache/skywalking-query-protocol
   2c59ae0..d6e2947  master     -> origin/master
 * [new tag]         v8.9.0     -> v8.9.0
 * [new tag]         v8.9.1     -> v8.9.1
 
 
  $ git rebase upstream/master
  $ git status
  On branch fix-istio-planes-naming
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   apm-protocol/apm-network/src/main/proto (new commits)
	modified:   oap-server/server-query-plugin/query-graphql-plugin/src/main/resources/query-protocol (new commits)
```

After rebasing those new "problematic" files showed up.

I couldn't `git restore` or `git checkout -- ` them, the only think that worked after googling was `git submodule update --init --recursive` 🤷  Do you know why @wu-sheng ? I am curious.
Anyhow fixed the PR and force pushed the changes.

wu-sheng commented:
When you face a submodule update in the upstream, your git(local) will prefer to keep your local version. So, in this time, you need to revert the local change.
